api-references/supervisely-annotation-json-format/point-cloud-episodes.md

```diff
@@ -172,17 +195,17 @@ Main idea of `key` fields and `id` you can see below in [Key id map file](point-
 * `geometryType` - "cuboid\_3d" -class shape
 * `geometry` - geometry of the object
 
-**Description for `geometry` field:**
+**Description for `geometry` field (cuboid\_3d):**
 
-* `position` 3D vector X, Y, Z values matches the axes on world coordinates, defined in global frame of reference as:
-  * **+x** - forward in the direction of travel ego vehicle
-  * **+y** - left
-  * **+z** - up
+* `position` 3D vector of box center coordinates:
+  * **x** - forward in the direction of the object
+  * **y** - left
+  * **z** - up
 * `dimensions` is 3D vector with:
```

Comment:
`dimensions` is a 3D vector that scales a cuboid from its local center along x,y,z